week1/task1/style.css

```
@@ -0,0 +1,198 @@
+header {
+  position: fixed;
+  width: 100%;
+  height: 10%;
+  background-color: #a3aa7d;
+  font-family: "Dongle", sans-serif;
+  border-radius: 20px;
+}
+
+header .container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.container h1 {
+  margin-left: 30px;
+  color: #ffff;
+  font-size: 30px;
+}
+.nav--icon {
+  margin-right: 30px;
+  color: #ffc585;
+}
+
+.nav--icon:hover .right--category__list {
+  display: block;
+}
+
+.right--category__list {
+  display: none;
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 150px;
+  height: 100vh;
+  margin-top: 8px;
+  border-radius: 10px;
+  color: #1a275a;
+  background-color: #a9c4df;
+  font-size: 30px;
+}
+
+.right--category__list p:hover {
+  color: white;
+}
+
+.wrapper {
+  display: flex;
+}
+
+.nav--left {
+  background-color: #fcd9a2;
+  width: 400px;
+  border-radius: 20px;
+  font-family: "IBM Plex Sans KR", sans-serif;
```

header 빼고 다 동일한 폰트인 것 같은데 *이나 body에 넣는건 어떨까요?!?

```suggestion
* {
  font-family: "IBM Plex Sans KR", sans-serif;
  }
```